src/SelectLocation.js

```diff
@@ -368,6 +369,10 @@ export default class SelectLocation extends Component {
 
       switch (data.message) {
         case "SIGNS_API_REQUEST":
+          // Use workOrderDetailsViewable state to disable red pin and pulse if user is Viewer role
+          if (data.view === "view_2619") {
```

Comment:
Nice, this is pretty clever. Kinda of a hard-coded hack but let's count our wins and never talk about this again.